docs/data-sources/resource_policies.md

```diff
@@ -0,0 +1,142 @@
+# Data Source: mongodbatlas_resource_policies
+
+`mongodbatlas_resource_policies` returns all resource policies in an organization.
+
+-> **NOTE**: Resource Policies are currently in Public Preview. You will need to
```

Comment:
```suggestion
-> **NOTE**: Resource Policies are currently in Public Preview. To use this feature, you must take the following actions:
```

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2